src/views/microworld/art/art.json

```diff
@@ -6,121 +6,123 @@
-            {
-               "type":"tip",
-               "title":"First, select a sprite",
-               "thumbnailUrl":"/images/microworlds/hiphop/dancer-sprite.png"
-            },
-            {
-               "type":"tip",
-               "title":"Then, try this script",
-               "thumbnailUrl":"/images/microworlds/hiphop/switch-wait.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Start it!",
-               "thumbnailUrl":"/images/microworlds/hiphop/green-flag.gif"
-            }
-         ]
-      },
-      {
-         "title":"Repeat the Dance",
-         "thumbnails":[
-            {
-               "type":"tip",
-               "title":"Add another \"wait\"",
-               "thumbnailUrl":"/images/microworlds/hiphop/add-wait.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Drag this block over",
-               "thumbnailUrl":"/images/microworlds/hiphop/add-repeat.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Start it",
-               "thumbnailUrl":"/images/microworlds/hiphop/green-flag.gif"
-            }
-         ]
-      },
-      {
-         "title":"Play Music",
-         "thumbnails":[
-            {
-               "type":"tip",
-               "title":"Add music that repeats",
-               "thumbnailUrl":"/images/microworlds/hiphop/add-play-sound.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Start it",
-               "thumbnailUrl":"/images/microworlds/hiphop/green-flag.gif"
-            }
-         ]
-      },
-      {
-         "title":"Shadow Dance",
-         "thumbnails":[
-            {
-               "type":"tip",
-               "title":"Add this block",
-               "thumbnailUrl":"/images/microworlds/hiphop/shadow-dance.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Start it",
-               "thumbnailUrl":"/images/microworlds/hiphop/green-flag.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Click the stop sign to reset",
-               "thumbnailUrl":"/images/microworlds/hiphop/stop.gif"
-            }
-         ]
-      },
-      {
-         "title":"More things to try",
-         "thumbnails":[
-            {
-               "type":"tip",
-               "title":"Try different dance moves",
-               "thumbnailUrl":"/images/microworlds/hiphop/change-dance-moves.gif"
-            },
-            {
-               "type":"tip",
-               "title":"Add more moves",
-               "thumbnailUrl":"/images/microworlds/hiphop/long-dance-script.png"
-            },
-            {
-               "type":"tip",
-               "title":"Change the timing",
-               "thumbnailUrl":"/images/microworlds/hiphop/change-dance-timing.png"
-            }
-         ]
-      }
-   ],
+   "commentedout_content": {
```

Comment:
@morantsur where is `commentedout_content` used? I'm having trouble finding it, sorry!